docs/killing-processes.md

```diff
@@ -0,0 +1,9 @@
+# Finding & Killing Processes
+
+It's very annoying when you're trying to run your postgres server on docker-compose but you can't start it because the port is already in use. To fix this:
+
+## Mac OS
```

Comment:
What if I am running it on a Windows laptop?

Reply:
It's similar but I have to look it up. We'll cross that bridge when we get there!